graph/step.go

```diff
@@ -347,3 +354,12 @@ func addBuildCacheOptsToCmd(domain, path, tag, originalBuildCmd string) (string,
 	}
 	return fmt.Sprintf("--load --cache-to=type=registry,ref=%s,mode=max --cache-from=type=registry,ref=%s %s", cacheImage.String(), cacheImage.String(), originalBuildCmd), nil
 }
+
+func invokesBuildkit(envs []string) bool {
+	for _, env := range envs {
+		if env == BUILDKIT_ENV_VAR {
```

Comment:
If there is space in the env string like  `DOCKER_BUILDKIT = 1`, can the code handle it?

Reply:
if there's space in the env string, then the Task itself will fail before running docker build, since I believe we just paste all the Env passed to `docker build` and any space in `--env` leads to error.

Reply:
For ex.

```sh
2021/06/01 20:35:23 Step args: /bin/sh, -c, docker run --rm --network acb_default_network --name build --volume source:/workspace --volume /var/run/docker.sock:/var/run/docker.sock \
       --volume home:/acb/home --env HOME=/acb/home --env DOCKER_BUILDKIT = 1 --workdir /workspace docker build -t samashahtest.azurecr.io/acb:blah -f test_Dockerfile .
2021/06/01 20:35:23 Launching container with name: build
docker: invalid reference format.
See 'docker run --help'.

```